docs/admin/garbage-collection.md

```diff
@@ -72,4 +72,4 @@ Including:
 | `--low-diskspace-threshold-mb` | `--eviction-hard` or `eviction-soft` | eviction generalizes disk thresholds to other resources |
 | `--outofdisk-transition-frequency` | `--eviction-pressure-transition-period` | eviction generalizes disk pressure transition to other resources |
 
-See [kubelet eviction design doc](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/admin/out-of-resource.md) for more details.
+See [Configuring Out Of Resource Handling](https://kubernetes.io/docs/admin/out-of-resource/) for more details.
```

Comment:
Change the URL to use a relative path, i.e.: 

\[Configuring Out Of Resource Handling](/docs/admin/out-of-resource/)